File: 백준/Bronze/1259. 팰린드롬수/팰린드롬수.swift
while true {
    let arr = Array(readLine()!)
    if arr == ["0"] { break }
    let count = arr.count
    var flag = true
    for i in (0..<count / 2) {
        if arr[i] != arr[count - i - 1] { 
            flag = false
            break
        }
    }
    print(flag ? "yes" : "no")
}

File: 백준/Bronze/1259. 팰린드롬수/팰린드롬수.py
while True:
    arr = input()
    count = len(arr)
    if arr == "0":
        break
    flag = True
    for i in range(0, count // 2):
        if arr[i] != arr[count - i - 1]:
            flag = False
            break
    print("yes" if flag else "no")